### DISCUSSION TITLE
Page not loaded after navigate from nova:tool pages

### DISCUSSION THREAD

**Discussion by sulfanoaf-cata** [laravel/nova-issues#6547] [Q&A]
Hi, 
I create many pages with Nova Tools
but i am facing a problem
after enter the page that made with Nova Tools, and then switch another page. the page is not loaded
no error on browser console and browser network.

How to fix this?

**Comment by nikoo-eshratabadi** [comment]
@sulfanoaf-cata i'm facing the same issue, can you please share your component?
